financial-connections/src/main/java/com/stripe/android/financialconnections/repository/FinancialConnectionsManifestRepository.kt

```diff
@@ -105,7 +129,8 @@ private class FinancialConnectionsManifestRepositoryImpl(
                     url = getManifestUrl,
                     options = options,
                     params = mapOf(
-                        NetworkConstants.PARAMS_CLIENT_SECRET to configuration.financialConnectionsSessionClientSecret
+                        NetworkConstants.PARAMS_CLIENT_SECRET to configuration.financialConnectionsSessionClientSecret,
+                        "expand" to listOf("active_auth_session")
```

Comment:
This makes sure we receive the entire auth session inside the manifest. We call this endpoint if process dies and we need to re-fetch the entire session state.